python/fledge/services/core/api/health.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+# FLEDGE_BEGIN
+# See: http://fledge-iot.readthedocs.io/
+# FLEDGE_END
+
+import logging
+import asyncio
+import json
+
+from aiohttp import web
+from fledge.common import logger
+from fledge.common.common import _FLEDGE_DATA
```

Comment:
import _FLEDGE_ROOT too and then use
 data_dir_path = _FLEDGE_DATA if _FLEDGE_DATA else _FLEDGE_ROOT + '/data'